roles/galera/defaults/main.yml

```diff
@@ -7,6 +7,8 @@ galera_bootstrap_node: ""
 mariadb_cluster_user: galera_sst_user
 mariadb_backup_user: backup_user
 
+percona_url_rpm: https://mirrors.cloud.tencent.com/percona/release/7/RPMS/noarch/percona-release-1.0-25.noarch.rpm
```

Comment:
Why not use the official Percona YUM repo?

Reply:
The url that is part of the role didn't work, it gave a 404.

Reply:
Or maybe use this one? 

https://repo.percona.com/yum/percona-release-latest.noarch.rpm

Mirror RPM URLs tend to disappear ...

Reply:
Yes, I am testing this locally now. Will commit and push with a minute. 👍